administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -188,7 +211,7 @@ public function getCheckForSelfUpdate()
         try {
             // Get the component extension ID
             $joomlaUpdateComponentId = $db->loadResult();
-        } catch (\RuntimeException $e) {
+        } catch (RuntimeException $e) {
```

Comment:
```suggestion
        } catch (\RuntimeException $e) {
```
Revert this change because in J4 the file is namespaced.